copilot/submit/manifest.yml

```diff
@@ -47,6 +47,7 @@ secrets:
   SECRET_KEY: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/SECRET_KEY
   RSA256_PUBLIC_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/RSA256_PUBLIC_KEY_BASE64
   ADDITIONAL_EMAIL_LOOKUPS: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/ADDITIONAL_EMAIL_LOOKUPS
+  NOTIFY_API_KEY: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/gov_notify_api_key_test
```

Comment:
The secret name shouldn't have 'test' in it, as this will make it inconsistent between environments, please can we rename it: `NOTIFY_API_KEY` to be consistent with the env var on the app?

Reply:
ah I meant more that it was the test key rather than in reference to the environment. But if we wanted to change it to a different key we'd just modify the secret rather than making a code change here to point to a different one. I'll rename

Reply:
changed this to `NOTIFY_API_KEY` and set for both test and dev